### ISSUE TITLE
Event has wrong DDL

### ISSUE THREAD

**Issue by kseniiaguzeeva** [dbeaver/dbeaver#10937]
#### System information: 
- DBeaver version 7.3.3

#### Connection specification:
- Database name and version MySQL
- 
#### Steps to reproduce, if exist:
1. Create the event:
```
CREATE EVENT myevent
    ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 1 HOUR
    DO
      UPDATE myschema.mytable SET mycol = mycol + 1;
```
2. Open the event and go to DDL tab.

Expected: 
1. the DDL is similar to what was used for creation
2. the DDL can be successfully run

Actual result:
1. the DDL is 
```
CREATE EVENT myevent
ON COMPLETION NOT PRESERVE
ENABLE
DO UPDATE myschema.mytable SET mycol = mycol + 1
```
2. the DDL execution fails

**Comment by LonwoLonwo** [comment]
"SHOW CREATE EVENT" can be used.

**Comment by LonwoLonwo** [comment]
Fixed in 21.0.0
Default DDL for UI-created event fixed.

**Comment by uslss** [comment]
verified
